charts/spire/charts/spire-server/values.yaml

```diff
@@ -96,6 +96,26 @@ federation:
     port: 8443
     address: "0.0.0.0"
 
+  ingress:
+    enabled: false
+    className: ""
```

Comment:
Maybe have this be `spire-federation` or something similar?

Reply:
For enabled?

Its hard to read in the patch, but its already spire-server.federation.ingress.enabled.

Reply:
<img width="330" alt="image" src="https://github.com/spiffe/helm-charts/assets/694733/39fc05f6-555b-44d2-9488-9d6cdf4d1a67">

When reviewing use these buttons to see more of the context. to better judge.

Reply:
referring specifically to the className default value

Reply:
ClassName is an ingress property. Unrelated to this chart. Needs to be configurable depending on what ingress classes are available on the users cluster.

Reply:
Yes, so do we want to give it a default value other than `""`?

Reply:
No. Blank should be default. If blank, no ingress class will be specified. This allows it to function properly on clusters where the cluster-admin has specified a default ingress class for the cluster for the users. Actual class names are not standardized so we can't set it to anything in particular thats useful.

Some details here:
https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class